app/views/distributions/leap-15.0.html.erb

```diff
@@ -108,11 +108,16 @@
           </div><!-- /.card -->
         </div><!-- /.card-deck -->
       </div><!-- /.tab-pane -->
-
     </div><!-- /.tab-content -->
-
+    <div>
```

Comment:
what about adding it some padding? 

I like this more here than where it was before :+1: however, in favour of the UX consistent principle, maybe we should add this message in the old version as well (even if we have the flash message). If you are in the new version page, go to the old one and then you want to go back, you would expect a link in the same place. :wink:

Reply:
if you just want some padding or margin, use this https://getbootstrap.com/docs/4.1/utilities/spacing/